frontend/src/metabase/common/components/FilterPicker/types.ts

```diff
@@ -1,10 +1,15 @@
-import type { ColumnMetadata, FilterClause, Query } from "metabase-lib/types";
+import type {
+  ColumnMetadata,
+  FilterClause,
+  ExpressionClause,
+  Query,
+} from "metabase-lib/types";
 
 export interface FilterPickerWidgetProps {
   query: Query;
   stageIndex: number;
   column: ColumnMetadata;
   filter?: FilterClause;
-  onChange: (filter: FilterClause) => void;
```

Comment:
I'm not 100% clear on where we landed on `FilterClause` v `ExpressionClause` - I think I understand the difference, but the naming is a little confusing.